### PR TITLE
All claims usability updates

### PR DESCRIPTION
**Pull request by cvalarida**
## Description
Changes the new disability pages.

## Testing done
Manual & unit tests.

## Screenshots
image
image

## Acceptance criteria
- [x] The pages show the content as given in the ticket
- [x] The page for adding a new disability is displayed only if the question on the first page is answered "Yes"

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/disability-benefits/all-claims/pages/addDisabilities.js [hunk 1]
```diff
@@ -44,43 +35,31 @@ export const uiSchema = {
...
-            <ul>
-              <li>The part of your body that's affected</li>
-              <li>
-                If your disability is on the right side or left side of your
-                body
-              </li>
-              <li>The part of your body that isn't working right</li>
-            </ul>
-          </AlertBox>
+          <div>
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/pages/addDisabilities.js]: Don't we usually put this kind of thing in the content directory?

File: src/applications/disability-benefits/all-claims/pages/addDisabilities.js [hunk 2]
```diff
@@ -44,43 +35,31 @@ export const uiSchema = {
...
+            <p>For example, foot pain, back pain, or hearing loss.</p>
+            <AdditionalInfo triggerText="What if I don't know the name of my condition?">
```
Comment (markgreenburg) [inline src/applications/disability-benefits/all-claims/pages/addDisabilities.js]: Looks like maybe the wrong kind of apostrophe here

**Review by markgreenburg — COMMENTED** [review]
Looks good; I think there might be a stray 'bad' apostrophe in the description content.

**Comment by peggygannon** [comment]
@cvalarida Hi, see below for a few content tweaks from me: 

1. for the first screen,
The content should be the below (we removed 'service-connected' per stakeholder feedback)
{first screen}
Do you have any new conditions you want to add to your claim?(*Required)
{radio}
Yes
No

2. for the second screen, I think the footnote looks odd and gets lost below the text field. Can we try it this way? 
**New condition**
If you know the name of your condition, you can type it here. You can write whatever you want and we'll make suggestions for possible disabilities (for example, foot pain, back pain, or hearing loss). (*Required)

{text field with matching}

File: src/applications/disability-benefits/all-claims/pages/addDisabilities.js (hunk 1 above)
Comment (cvalarida) [reply src/applications/disability-benefits/all-claims/pages/addDisabilities.js]: Pfffffft. Yeah. You're right. Good call.

File: src/applications/disability-benefits/all-claims/pages/addDisabilities.js (hunk 2 above)
Comment (cvalarida) [reply src/applications/disability-benefits/all-claims/pages/addDisabilities.js]: :male_detective:

**Comment by cvalarida** [comment]
@peggygannon Updated screenshots:
image
image

**Review by markgreenburg — APPROVED** [review]
:shipit:

**Comment by peggygannon** [comment]
@cvalarida Thanks.. for the first screen, the question should be:  Do you have any new conditions you want to add to your claim?(*Required)

we need to remove 'disabilities' from this question

**Comment by cvalarida** [comment]
image

**Review by peggygannon — APPROVED** [review]
thanks, @cvalarida Looks good!

**Pull request merged**